wpeverest/everest-forms
# #781 Tweak - Change URL with new website URL

### All Submissions:






### Changes proposed in this Pull Request:



The purpose of this PR is to change the existing URL with a new website URL. Please verify all the links and documentation link across the plugins is redirecting to specific pages and should not be broken or not found.

### How to test the changes in this Pull Request:

1. Checkout to this branch and verify all links are redirectable to the intended pages. 

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Change all URL with the new website URL
